elytron/src/main/java/org/wildfly/extension/elytron/_private/ElytronSubsystemMessages.java

```diff
@@ -697,6 +697,11 @@ public interface ElytronSubsystemMessages extends BasicLogger {
     @Message(id = 1211, value = "Unable to load the credential store.")
     OperationFailedException unableToLoadCredentialStore(@Cause Throwable cause);
 
+    @Message(id = 1212, value = "KeyStore does not contain a PublicKey and PrivateKey for KeyStore: [%s] and alias: [%s].")
+    StartException invalidKeyPair(String keyStore, String alias);
+
+    @Message(id = 1213, value = "Unable to update the filesystem realm with the new keystore: %s")
+    OperationFailedException unableToUpdateFilesystemKeystore(@Cause Exception cause, String causeMessage);
```

Comment:
Maybe we could update this message to make it more clear, e.g., maybe something like "Unable to update the filesystem realm to make use of the provided key pair for integrity" or something along those lines.